azure-proto-core/ResourceOperationsBase.cs

```diff
@@ -9,9 +9,9 @@ namespace azure_proto_core
     /// TODO: Split into ResourceOperations/TrackedResourceOperations
     /// </summary>
     /// <typeparam name="Model"></typeparam>
-    public abstract class ResourceOperationsBase<TOperations, TResource> : OperationsBase
-        where TResource : Resource
-        where TOperations : ResourceOperationsBase<TOperations, TResource>
+    public abstract class ResourceOperationsBase<TOperations, TResource> : DeletableResourceOperations <TResource, TOperations>
+        where TResource:Resource 
+        where TOperations: DeletableResourceOperations<TResource, TOperations>
```

Comment:
Does this mean that a resource that has tags has to be deletable?  Is that always the case?